spk/kavita/src/appsettings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "TokenKey": "super secret unguessable key that is longer because we require it",

```

Comment:
please generate a unique TokenKey at installation time.

If the user must know this token, please show it once in the installation wizard.

Reply:
When the app is first run it generates that key. This is the default file that's present in the source repo. I just used it to change the port number from 5000 to 5500 to avoid a conflict with DSM.